### ISSUE TITLE
USWDS-Site: Create "This is what's new" intro post

### ISSUE THREAD

**Issue by amyleadem** [uswds/uswds-site#2930]
## Summary
Create a post about rebuilding the news page.

**Comment by finekatie** [comment]
Hey @amyleadem , I don't know that we can do a post or blurb about this. We wouldn't be linking to anything unless someone was going to write a whole 500 word article. That seems unnecessary at this time. I'm in favor of simply getting the page up and getting it mentioned in our newsletter, call and elsewhere. Sound good? If so, I think we can close this task.

**Comment by finekatie** [comment]
Recommended not doing this unless we add it to the newsletter or other place. @brunerae do I keep this for next sprint for adding to newsletter?

**Comment by finekatie** [comment]
Hi @annepetersen and @jdotyoon - I'm thinking about closing this as "not doing", as we won't need to make an announcement about it on its own page. This will be something we add to Jan. newsletter. Thoughts?

**Comment by jdotyoon** [comment]
I agree if we're going to add it in the Jan newsletter!

**Comment by annepetersen** [comment]
Makes sense. Do we have a plan for a different post to restart publications on that page?

**Comment by finekatie** [comment]
> Makes sense. Do we have a plan for a different post to restart publications on that page?

Programmatically, making an announcement as a blurb on the page with no page to link to would require removing link capability from the header
![Image](https://github.com/user-attachments/assets/eb29eb6f-4e1e-42da-b129-be9e84a84b1f)

It can be done, but do we want to?

I think it makes the most sense to simply start adding the [newest content](https://docs.google.com/document/d/1nMJIyzaYyh_IE7Isc5Q2qni-Khwk4duKce8SkR2zB5U) into the feed. It will push out the old content to the "All news and updates" page, and push it further down from sight for the time being. 

I recommend announcing the page in the Newsletter and Monthly call in January. I think we can also promote it on Slack. 

I think if there is a strong desire to announce/alert on the page, perhaps some text at the top in a summary box or something that we can remove in 6-8 months. 

![Image](https://github.com/user-attachments/assets/3ebc5822-8624-4d15-8510-2c1bc0812364)

That way, it doesn't interfere with the flow of news coming in.

That's my two cents for the time being. Look forward to discussing when I get back.

**Comment by dct2023** [comment]
Confirmed with the team that this can be moved to 'Decided Against'.

**Comment by finekatie** [comment]
Hey @dct2023 , this is still in my queue as On Hold. The Status label doesn't take it out. Can I check off as Blocked or Done since there's no actual Status item for a task we're not doing?

**Comment by finekatie** [comment]
Hey @jdotyoon and @dct2023 , can this be closed since we are not doing this?

**Comment by annepetersen** [comment]
Wasn't tagged, but yes, yes we can
